src/tacacs/bash_tacplus/bash_tacplus.c

```diff
@@ -224,20 +227,40 @@ int tacacs_authorization(
     return result;
 }
 
+/*
+ * Get current SSH session remote address from environment variable
+ */
+void get_remote_address(char* dst, size_t size)
+{
+    // SSHD will create environment variable SSH_CONNECTION after user session created.
+    char *ssh_connection = getenv("SSH_CONNECTION");
```

Comment:
>SSH_CONNECTION [](http://example.com/codeflow?start=35&length=14)

SSH_CONNECTION include 4 parts: client IP address,
client port number, server IP address, and server port number. Do you need just one of them?

Reply:
Fixed